# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mysqlcc/fixed_mysql_4.1.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./shared/src/CMySQL.cpp.orig	2005-02-12 13:06:13.000000000 -0300
+++ ./shared/src/CMySQL.cpp	2005-02-12 13:07:25.842117042 -0300
@@ -451,7 +451,7 @@
   if (!isConnected())
     return false;
   
-  if (mysql_shutdown(mysql) != 0)
+  if (mysql_shutdown(mysql, SHUTDOWN_DEFAULT) != 0)
   {
     if (emiterror)
       emitError();
